eno-core/src/main/java/fr/insee/eno/core/processing/out/steps/lunatic/calculatedvariable/ShapefromAttributeRetrievalFromVariableGroups.java

```diff
@@ -0,0 +1,23 @@
+package fr.insee.eno.core.processing.out.steps.lunatic.calculatedvariable;
+
+import fr.insee.eno.core.model.EnoQuestionnaire;
+import fr.insee.eno.core.model.variable.Variable;
+import fr.insee.eno.core.model.variable.VariableGroup;
+
+import java.util.Optional;
+
+public class ShapefromAttributeRetrievalFromVariableGroups implements ShapefromAttributeRetrieval {
+
+    public Optional<Variable> getShapeFrom(String lunaticCalculatedVariableName, EnoQuestionnaire enoQuestionnaire) {
```

Comment:
A bit of javadoc here would be helpful (not easy to get the purpose of this without a close look, at least to me 😅 )